Doc 1766/changelog (#638)
swan-io/docs: changelog/2026/2026-05-07.mdx

---
title: 7 May 2026
type: release
---

## Improvements

🧭 Starting **June 4**, [Standing Orders](/topics/payments/credit-transfers/sepa/guide-schedule-so/) execute on their `lastExecutionDate` value. For example, a Standing Order with `lastExecutionDate` set to January 10 now runs on January 10 itself.

> 🛡️ Starting **May 25**, Single Device Authentication rolls out gradually to all OAuth2 flows. The clickable SMS link is replaced by a 6-digit OTP code across all partner integrations, [matching the approach already live on Swan's web banking](/changelog/26-february-2026/#improvements). This closes a remaining authentication fraud vector for partners using their own `client_id`.

Reviewer: ```suggestion
🛡️ Starting **May 25**, Single Device Authentication rolls out gradually to all OAuth2 flows. A 6-digit verification code replaces the clickable SMS link across all partner integrations, [matching the approach already live on Swan's web banking](/changelog/26-february-2026/#improvements). This closes a remaining authentication fraud vector for partners using their own `client_id`.
```

Active.

For your end users signing in or signing up from desktop:

> - They receive a 6-digit OTP code by SMS, replacing the previous clickable link.

Reviewer: We've been using "verification code" in our other references to the OTP.

- They enter the code on desktop to complete the flow.

For [identification](/topics/users/identifications/), two text messages are sent: one with a redirection link, and one to trigger identification on mobile.

No code changes are required, but please update your support documentation, guides, and flow screenshots accordingly.

## API updates

### Upcoming breaking changes

> ⚠️ On **June 4**, `Card`-named types and fields used for [accepting online card payments](/topics/merchants/online/cards/) are removed. The new `OnlineCard`-named equivalents are already in the schema; update your queries and mutations before this date.

Reviewer: ```suggestion
⚠️ On **June 4**, we're removing `Card`-named types and fields used for [accepting online card payments](/topics/merchants/online/cards/). The new `OnlineCard`-named equivalents are already in the schema; update your queries and mutations before this date.
```
Active and names the actor - i.e. us.

Reviewer: Also unsure about card-named types? Can we not just say we're replacing card with onlinecard in all places where its used?

Reviewer: Since we call it out further down

> ⚠️ On **June 4**, `Card`-named types and fields used for [accepting online card payments](/topics/merchants/online/cards/) are removed. The new `OnlineCard`-named equivalents are already in the schema; update your queries and mutations before this date.

Reviewer: Try this instead: 

⚠️ On **June 4**, Swan renames `Card`-prefixed merchant types and mutations to `OnlineCard` equivalents for [accepting online card payments](/topics/merchants/online/cards/). The deprecated names are removed on this date; update your queries and mutations before then.

Reviewer: This then makes the following content easier to scan - vs repeating deprecated multiple times.

This rename clarifies the distinction between online card acceptance and in-person card acceptance (terminals, Tap to Pay). It only affects partners accepting payments through [merchant profiles](/topics/merchants/profiles/guide-request/). Card issuing is unchanged.

**Queries:**

> - [`CardMerchantPaymentMethod`](https://api-reference.swan.io/objects/card-merchant-payment-method/) is deprecated; use [`OnlineCardMerchantPaymentMethod`](https://api-reference.swan.io/objects/online-card-merchant-payment-method/) instead. This affects all queries that resolve `CardMerchantPaymentMethod` from `MerchantProfile.merchantPaymentMethods` or [`MerchantPaymentMethod`](https://api-reference.swan.io/interfaces/merchant-payment-method/).

Reviewer: So its deprecated now - and will be removed 4 June? Have we mentioned the deprecation before?

Author: This is the first changelog mention of the deprecation. The rewrite above frames it as a rename happening on June 4 rather than relying on "deprecated" wording, which avoids implying a prior announcement.
